doc/api/globals.markdown

```diff
@@ -143,8 +143,6 @@ when to use `module.exports`.
 
 See the [module system documentation][] for more information.
 
-See the [module section][] for more information.
-
```

Comment:
This change is incomplete. The corresponding `[module section]: modules.html` has to be removed as well, as it is obsolete now.